TicketSwap/solar
# Make it possible to normalize the combobox results

# Description

This PR will make it possible to remove accentuated characters from a string and replace them by the normal equivalent. It will also implement this feature for the results of the Combobox. See test/storybook to see the result 

Context:
https://github.com/TicketSwap/sirius/issues/3709

## Changes

- [x] Added normalizeChars util
- [x] Added test for this
- [x] Added `normalize` prop to combox that will normalize the results of the combobox
- [x] Added 'Normalize' story for this

## How to test

- Checkout this branch
-  `yarn dev`
- `yarn storybook`
- Checkout the story 'Normalize' on storybook
- `yarn test` to see that the tests are passing
